app/services/block_domain_service.rb

```diff
@@ -3,12 +3,34 @@
 class BlockDomainService < BaseService
   def call(domain_block)
     if domain_block.silence?
-      Account.where(domain: domain_block.domain).update_all(silenced: true)
```

Comment:
I like the change to move this to smaller methods.  Possible further refactor...

- Set `@domain_block` accessor right in call, and then call a method like `process_block` or something, and do nothing else in `call`
- `process_block` would have the if/else logic around silence vs suspend
- Add a method like `blocked_domain` which is `domain_block.domain` and replace a few of those
- Add something like `domain_accounts` which does the Account.where(...)` that's repeated a few times here